feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-bundle-dropdown/configurator-attribute-single-selection-bundle-dropdown.component.ts

```diff
@@ -21,8 +23,18 @@ export class ConfiguratorAttributeSingleSelectionBundleDropdownComponent
   attributeDropDownForm = new FormControl('');
   selectionValue: Configurator.Value;
 
+  /**
+   * @deprecated since 4.1: remove redundant input parameter
+   */
   @Input() group: string;
 
+  constructor(
```

Comment:
we can just use the parent's constructor, right?

Reply:
Done!